src/database/sql/core/offset.sql

```diff
@@ -0,0 +1,9 @@
+-- selectStableTransactionOffsetById
```

Comment:
Minor style nit: I would call the file `offsets.sql` for consistency.

Reply:
done